src/sugar3/power.py

```diff
@@ -15,6 +15,17 @@
 # Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 # Boston, MA 02111-1307, USA.
 
+"""The power module provides an interface to *powerd*, a daemon that
```

Comment:
As I've seen, most of new documentation is written after the 
`"""`
can you change it?